cli/tests/test_parallelize_command.rs

```diff
@@ -305,9 +325,18 @@ fn test_parallelize_interior_target_is_a_merge() {
     ◉  000000000000
     "###);
 
-    insta::assert_snapshot!(test_env.jj_cmd_failure(&workspace_path,&["parallelize", "description(1)::"]),
-        @r###"
-    Error: Only the roots of the target revset are allowed to have parents which are not being parallelized.
+    test_env.jj_cmd_ok(&workspace_path, &["parallelize", "description(1)::"]);
+    insta::assert_snapshot!(get_log_output(&test_env, &workspace_path), @r###"
+    @    cd0ac6ad1415 3
+    ├─╮
+    │ │ ◉  1c240e875670 2
+    ╭─┬─╯
```

Comment:
nit: maybe better to use `get_log_output_with_parents()` globally? It's uneasy to read the ancestry from the graph.

Reply:
Done in a commit just before this commit.